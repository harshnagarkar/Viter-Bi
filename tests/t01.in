3 2 10
0.3333334 0.3333333 0.3333333
0.45 0.52 0.25
0.35 0.3 0.13
0.2 0.18 0.62
0.5 0.5
0.85 0.15
0.1 0.9
1 0 1 0 1 1 0 0 0 1 
0 0 0 0 1 2 2 0 0 1 0 
