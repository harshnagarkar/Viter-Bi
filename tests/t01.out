20.450201802471 22.980716519170 21.132719612615 
1 0 1 0 1 0 0 1 0 1 0 
54.55
